spec/queries/my_facilities/registrations_query_spec.rb

```diff
@@ -48,22 +35,21 @@
           end
         end
 
-        specify { expect(registrations_query.registrations.count).to eq(included_timestamps.count) }
+        it 'counts registrations diagnosed with hypertension in the last_n quarters' do
```

Comment:
I think the specify is clean enough, no need to break it up and claim HTN explicitly